flink-contrib/flink-statebackend-rocksdb/src/test/java/org/apache/flink/contrib/streaming/state/RocksDBAsyncSnapshotTest.java

```diff
@@ -333,7 +339,7 @@ public void testCleanupOfSnapshotsInFailureCase() throws Exception {
 
 		RocksDBStateBackend backend = new RocksDBStateBackend(stateBackend);
 
-		backend.setDbStoragePath("file:///tmp/foobar");
+		backend.setDbStoragePath("file://" + temporaryFolder.newFolder().getAbsolutePath());
```

Comment:
To make this cross compatible with Windows (and other OSs), can you change that to `backend.setDbStoragePath(temporaryFolder.newFolder().toUri().toString());`